views/login/dateOfBirth.pug

```diff
@@ -7,11 +7,15 @@ block variables
   -var dobDay = hasData(body, 'dobDay') ? body.dobDay : (hasData(data.login, 'dobDay') ? data.login.dobDay : '')
   -var dobMonth = hasData(body, 'dobMonth') ? body.dobMonth : (hasData(data.login, 'dobMonth') ? data.login.dobMonth : '')
   -var dobYear = hasData(body, 'dobYear') ? body.dobYear : (hasData(data.login, 'dobYear') ? data.login.dobYear : '')
+  -var firstName = hasData(data, 'login.firstName') ? data.login.firstName : (hasData(data, 'personal.firstName') ? data.personal.firstName : '' )
 
 block content
 
   h1 #{title}
 
+  if firstName
+    h2 #{firstName}, #{__('thanks for your social insurance number')}.
```

Comment:
Let's capitalise the 'S' like on the open/close component.

![image](https://user-images.githubusercontent.com/2454380/70180525-35be3580-16ae-11ea-8b98-d539cc4e9df4.png)